projects/court-case-and-delius/src/dev/kotlin/uk/gov/justice/digital/hmpps/data/entity/Entities.kt

```diff
@@ -27,7 +27,7 @@ class AssessmentType(@Id val assessmentTypeId: Long, val description: String)
 class CaseAllocation(@Id val caseAllocationId: Long, val eventId: Long)
 
 @Entity
-class Court(@Id val courtId: Long, val courtName: String)
+class Court(@Column(name = "court_id") @Id val courtId: Long, val courtName: String)
```

Comment:
The dev sources aren't included in the packaged image, so I don't think this entity will be the one causing the problem in the pod.  Also the `courtId` field name would map to `court_id` for the column name anyway.

Reply:
i realised this after i pushed, and i am fixing this